sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/state/RocksDBStateStoreSuite.scala

```diff
@@ -1090,6 +1089,42 @@ class RocksDBStateStoreSuite extends StateStoreSuiteBase[RocksDBStateStoreProvid
     }
   }
 
+  test("verify that column family id is assigned correctly after removal") {
```

Comment:
could we add a test to ensure that vcf ids are reused correctly on restart ?

Reply:
Adding a test case in the TransformWithStateSuite might be tough at this point in time because we don't support removal/deleteIfExists and don't have access to the provider. We have existing test cases that do restart queries so we should be good as far as that goes.